### ISSUE TITLE
Error parsing runner response.

### ISSUE THREAD

**Issue by mr-const** [jenkinsci/saltstack-plugin#9]
I'm running runner's task filesystem.update, It completes OK, but it seems that plugin cannot parse it's response and marks build as failed.

Here is successful execution by curl:
```
D:\sources\android>curl -X POST https://mysalt.domain.tld/salt/ -H "X-Auth-Token: 12345" -H "Content-Type: application/json" -d "[{\"client\":\"runner\",\"mods\":\"\",\"tgt\":\"notused\",\"expr_form\":\"nodegroup\",\"fun\":\"fileserver.update\",\"arg\":[]}]"
{"return": [true]}
```

Here is failing log from jenkins:
```
Building remotely on dsg-buildman-mac-00 (android ios mac) in workspace /Users/jenkins/workspace/RS-Server-Deploy
Sending JSON: [{"client":"runner","mods":"","tgt":"notused","expr_form":"nodegroup","fun":"fileserver.update"}]
ERROR: Build step failed with exception
java.lang.ClassCastException: java.lang.Boolean cannot be cast to net.sf.json.JSONObject
	at com.waytta.Utils.validateFunctionCall(Utils.java:141)
	at com.waytta.SaltAPIBuilder.perform(SaltAPIBuilder.java:339)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:782)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:534)
	at hudson.model.Run.execute(Run.java:1738)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:410)
Build step 'Send a message to Salt API' marked build as failure
Finished: FAILURE
```

**Comment by mchugh19** [comment]
Thanks! Looks like we were not handling the case where a runner only returns a boolean and not json. 

I've worked up a version that now seems to work for me. Can you test this out and let me know how it goes? 
https://github.com/jenkinsci/saltstack-plugin/releases/tag/v1.6.1-beta

**Comment by mr-const** [comment]
I can confirm, it works as expected now, thanks.

**Comment by mchugh19** [comment]
Super!

In the just released 1.6.1 build. Should hit the jenkins mirrors and be available in the next 24 hours.
